iosApp/iosApp/iOSApp.swift

```diff
@@ -1,10 +1,26 @@
 import SwiftUI
+import Shared
+
+struct ContentViewNew: View {
+        // Create helper instance
+    let greet = GreetingHelper().greet()
+
+    var body: some View {
+        Text(greet)
+    }
+}
+
 
 @main
 struct iOSApp: App {
+
+	init() {
+	   HelperKt.doInitKoin()
+	}
+
 	var body: some Scene {
 		WindowGroup {
-			ContentView()
```

Comment:
Couldn't you just change the ContentView instead of creating a new one? I think the ContentView is not used anymore